tests/test_distro.py

```diff
@@ -9,4 +9,4 @@ def test_version():
     """
     Tutor distro plugin tests version
     """
-    assert __version__ == '15.0.0'
+    assert __version__ == '15.0.2'
```

Comment:
I think this test is going to be annoying to maintain. I think it would be good to eliminate it. What do you think?

Reply:
Sorry, I forget remove this, it is not useful

thanks, @MaferMazu